src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,96 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String DEFAULT_DELIMETER = ",";
+    private static final String KEY_SUPPLY = "supply";
+    private static final String KEY_BUY = "buy";
+    private int supply = 0;
+    private int buy = 0;
+    private int result = 0;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        clear();
+        readFromFile(fromFileName);
+        calculateResult();
+        writeToFile(toFileName);
+    }
```

Comment:
![image](https://github.com/mate-academy/jv-work-with-file/assets/156338484/7b003ba9-c9ba-46f9-abc3-dd75c0130c27)
better try this case